samples/GalleryApp/Pages/Skills.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "Name": "Vision",
```

Comment:
recommendation: you could rename this file to something more representative (i.e. SkillViewGlossary.json)